### PR TITLE
docs: remove jsdoc in favor of manually written docs

### LINKED ISSUE #45: Cleanup auto-magic docs
There are a few artifacts from the auto-magic docs from jsdoc-to-markdown that we should clean up and/or organize. We can customize the output of the docs via the README.hbs template.

https://github.com/jsdoc2md/jsdoc-to-markdown/wiki/Cherry-picking-which-documentation-appears-in-output

### PR DESCRIPTION
**Pull request by marksmccann**
Removed the jsdoc-2-markdown dependency in favor of manually written documentation. Removed jsdoc eslint plugin. Removed verbose comments from index.js. Updated prettier trailing comma option.

closes #45

### CODE REVIEW

File: .prettierrc.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-trailingComma: "es5"
+trailingComma: "none"
```
Comment (phillipluther) [inline .prettierrc.yml]: You don't like trailing commas?! I've loved them for a while for how easy it is to reorder properties/array items or comment stuff out, or add items.

It's one of the first of the first-world problems; not losing sleep over it.

File: README.md [hunk 2]
```diff
@@ -50,247 +48,106 @@ const results = sass.renderSync({
...
-
-## node-sass-extra
-The Node API for `node-sass-extra`.
+## Options
 
+|Name|Type|Description|
+|----|----|-----------|
+|data|`string` \| `string[]`|String(s) to be compiled.|
+|file|`string` \| `string[]`|File(s) to be compiled; can be a file path or a glob pattern.|
+|output|`string` \| `function`|The output destination; if provided, files WILL be written to disk. Can be a file path, a directory, or a callback that returns a file path or directory. Must be/return a file path when paried with `data`.|
```
Comment (phillipluther) [inline README.md]: "... a file path when _paired_ with `data`."

One more "paried" below, too.

File: .prettierrc.yml (hunk 1 above)
Comment (marksmccann) [reply .prettierrc.yml]: I can see that. There is just something about that dangling, unnecessary comma that really bothers me, especially when the only argument for including it is just a matter of occasional convenience. Maybe, I could be converted, but at this point, not a fan of trailing commas.

File: README.md (hunk 2 above)
Comment (marksmccann) [reply README.md]: Good catch, ill get this updated and and merged in.

**Pull request merged**